File: backend/java/elo-calculator-backend/src/main/java/de/iits/elocalculatorbackend/player/model/entity/Player.java
package de.iits.elocalculatorbackend.player.model.entity;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.Setter;

import java.util.UUID;

@Entity
@Table(name = "player")
@Getter
@Setter
public class Player {

    @Id
    @GeneratedValue(strategy = GenerationType.UUID)
    private UUID id;

    @Column(name = "user_name")
    private String username;

    @Column(name = "display_name")
    private String displayName;

    @Column(name = "email")
    private String email;

    @Column(name = "elo")
    private int elo = 100;

}

File: backend/springboot-kotlin/src/main/kotlin/de/iits/elo/player/Player.kt
package de.iits.elo.player

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "player")
data class Player(

    @Id
    @Column(name = "user_name")
    val userName: String,
    @Column(name = "display_name")
    val displayName: String,
    @Column(name = "email")
    val email: String?,
    @Column(name = "elo")
    val elo: Int = 1000,
)